### PR TITLE
AIRAC 2312 (Do not merge until 30/11)

### PR DESCRIPTION
**Pull request by alphadelta332**
## Summary
Changes to SOPs, Waypoint names, Procedures for AIRAC 2312

## Changes
**Fixes**:
- Moved BK ADC -> SY TCU to Departures Coordination section
- Minor typos in SY ADC page

**Changes**:
- Waypoint name changes
	- POONA > MUNAR
	- SWIFT > AKROM
	- CONNI > SAPKO
	- KEELS > OPEGA
	- WAVES > IPMOR
	- CLARK > RIBLI
	- IRONS > MIKIS
	- KAREN > VEKLO
	- NOLAN > VIMAP
- YSSY SID Selection changes
- YMLT SID Selection changes

### CODE REVIEW

File: docs/aerodromes/sydney.md [hunk 1]
```diff
@@ -275,7 +315,7 @@ Sydney Coordinator is activated when required to reduce frequency congestion on
 !!! example
     **VOZ543**: "Sydney Delivery, VOZ543, PDC read back"  
     **SY ACD**: "VOZ543, go ahead the read back"  
-    **VOZ543**: "ENTRA5 departure, squawk 1336, bay 33, VOZ543"  
+    **VOZ543**: "OLSEM5 departure, squawk 1336, bay 33, VOZ543"
```
Comment (mattkelly4) [inline docs/aerodromes/sydney.md:318]: Looks like its going to be the OLSEM1 departure

File: docs/terminal/sydney.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ c) Departures assumes **SRI** airspace when the position is inactive
...
+    The default ownership of sectors within the SY TCU is merely a suggestion for starters. There are 7 executive controller positions within the SY TCU, plus a flow controller, and the ownership of these sectors can be delegated as desired based on the traffic disposition, when agreed between the controllers. For example, during a Milk Run event, if SY APP and SY DEP are online, SY APP may have a lot more work to do than SY DEP, and it would mostly be concOLSEMted on the RIVET/ODALE corridor. In light of this, it might be wise for SY APP to take ownership of SAS, SFW and SFE airspace, whilst SY DEP take ownership of SAN, SDS, SDN, and SRI airspace.
```
Comment (mattkelly4) [inline docs/terminal/sydney.md:44]: > concOLSEMted

CTRL+F strikes again ;)

File: docs/terminal/sydney.md [hunk 3]
```diff
@@ -230,8 +230,8 @@ To reduce frequency congestion, several commonly used geographically defined are
 
 | Sector Name | Lateral Limits | Vertical Limits  |
 | --| ----------------| --------- |
-| City East | Bounded by Rushcutters Bay, Sydney Cricket Ground, Sydney Harbour Bridge South Pylon, Fort Denison, Clark Island, Rushcutters Bay   | Not above `A020`     |
-| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, Clarke Island, Rushcutters Bay | Not above `A020` |
+| City East | Bounded by Rushcutters Bay, Sydney Cricket Ground, Sydney Harbour Bridge South Pylon, Fort Denison, RIBLI Island, Rushcutters Bay   | Not above `A020`     |
```
Comment (mattkelly4) [inline docs/terminal/sydney.md:233]: > RIBLI Island

Another CTRL+F (should be Clark Island)

File: docs/terminal/sydney.md [hunk 4]
```diff
@@ -230,8 +230,8 @@ To reduce frequency congestion, several commonly used geographically defined are
...
+| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, RIBLIe Island, Rushcutters Bay | Not above `A020` |
```
Comment (mattkelly4) [inline docs/terminal/sydney.md:234]: As above: Clark Island

**Review by mattkelly4 — CHANGES_REQUESTED** [review]

File: docs/terminal/sydney.md (hunk 2 above)
Comment (LeGiraffe) [inline docs/terminal/sydney.md:44]: Hahahahaha

Comment (alphadelta332) [reply docs/terminal/sydney.md:44]: ahahahaha what a stitch up, good find

File: docs/terminal/sydney.md (hunk 4 above)
Comment (LeGiraffe) [inline docs/terminal/sydney.md:234]: Almost

**Review by LeGiraffe — COMMENTED** [review]
😂

File: docs/terminal/sydney.md (hunk 4 above)
Comment (alphadelta332) [reply docs/terminal/sydney.md:234]: RIBLI island has a ring to it

**Review by mattkelly4 — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

File: docs/terminal/sydney.md [hunk 5]
```diff
@@ -231,7 +231,7 @@ To reduce frequency congestion, several commonly used geographically defined are
 | Sector Name | Lateral Limits | Vertical Limits  |
 | --| ----------------| --------- |
 | City East | Bounded by Rushcutters Bay, Sydney Cricket Ground, Sydney Harbour Bridge South Pylon, Fort Denison, Clark Island, Rushcutters Bay   | Not above `A020`     |
-| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, Clarke Island, Rushcutters Bay | Not above `A020` |
+| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, Clark Island, Rushcutters Bay | Not above `A020` |
```
Comment (LeGiraffe) [inline docs/terminal/sydney.md:234]: Clarke Island

**Review by LeGiraffe — CHANGES_REQUESTED** [review]

File: docs/terminal/sydney.md (hunk 5 above)
Comment (alphadelta332) [reply docs/terminal/sydney.md:234]: ERSA has a typo, it says Clark Island on the first one, Clarke Island on the second one. Checked google maps and correct spelling is Clark Island

Comment (LeGiraffe) [reply docs/terminal/sydney.md:234]: Interesting Apple Maps said with an E... but appears you are correct 👍🏻

**Review by LeGiraffe — APPROVED** [review]

**Pull request merged**